fastlane/Helpers/Test.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Test.swift
+//  FastlaneRunner
+//
+//  Created by Khanh on 27/09/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+enum Test {
+
+    static func buildAndTest(
```

Comment:
let's allow passing device names (if it can be array would be best) to match [current ruby implementation](https://github.com/nimblehq/ios-templates/blob/5664de6fc4a9da259295c4c1a1d3841d69b1dd72/fastlane/Managers/TestManager.rb) 🙏

Reply:
I updated at https://github.com/nimblehq/ios-templates/pull/363/commits/8cc4f66e3fe2b3c2b42bcd876cbade6b7deb4393